addSubcmd.go

```diff
@@ -15,10 +15,12 @@ func addSubcmd(args []string) {
 	addCmd := flag.NewFlagSet("add", flag.ExitOnError)
 
```

Comment:
```suggestion
	addCmd.SetOutput(flag.CommandLine.Output())
```

Needs to be here otherwise one line escapes